src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1416,7 +1417,7 @@ class ApplicationManagerImpl
   // Thread that pumps messages audio pass thru to mobile.
   impl::AudioPassThruQueue audio_pass_thru_messages_;
 
-  HMICapabilities hmi_capabilities_;
+  std::auto_ptr<HMICapabilities> hmi_capabilities_;
```

Comment:
@vlantonov, you right. But, as I noticed, that it not so common situation in SDL where it's really needed (At least right now). So for now there no need in this.